CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## v1.0.6
+
+*Jun 17, 2022*
+
+* Improve behavior of KMS in Ostracon
+* Improve supporting build environment
+  * Support building/running darwin/arm64
+  * Support building/running of linux/arm64 on Local/Docker via darwin/arm64
+  * Stop supporting linux/arm(32bit)
+
+### BREAKING CHANGE
+- Nothing
+
+### FEATURES
+- Nothing
+
+### IMPROVEMENTS
+- [kms] [\#417](https://github.com/line/ostracon/pull/417) Add KMS functionality
+  [build] [\#426](https://github.com/line/ostracon/pull/426) Remove binary check for localnode
+- [build] [\#431](https://github.com/line/ostracon/pull/431) Stop support for linux/arm(32bit)
```

Comment:
Shouldn't stopping support for ARM (32-bit) be a BREAKING CHANGE since it relates to product spec?

Reply:
OK, I'll update it.